### PR TITLE
Passing final comments

### PR DESCRIPTION
**Pull request by HussainGynai**
## Description
Continuation of the [new passing pr](https://github.com/RoboJackets/robocup-software/pull/1644), just fixing some remaining suggestions that were made.

### CODE REVIEW

**Review by kylestach — APPROVED** [review]
LGTM

**Review by Yudai-8-om — APPROVED** [review]
LGTM!

File: rj_gameplay/rj_gameplay/action/pivot.py [hunk 1]
```diff
@@ -31,9 +31,9 @@ def tick(self, intent: RobotIntent) -> None:
 
     def is_done(self, world_state:rc.WorldState) -> bool:
         #TODO: Change this when we get action state feedback
-        angle_threshold = 0.1
+        angle_threshold = np.pi / 180 #One dgree
```
Comment (kfu02) [inline rj_gameplay/rj_gameplay/action/pivot.py:34]:
```suggestion
        angle_threshold = np.pi / 180 #One degree
```
Small typo in a useful comment

File: rj_gameplay/rj_gameplay/tactic/pass_tactic.py [hunk 2]
```diff
@@ -61,8 +61,8 @@ def __init__(self, target_point:np.ndarray):
         self.target_point = target_point
         self.pivot_kick = tactic.SkillEntry(pivot_kick.PivotKick(target_point = target_point, chip=False, kick_speed=4.0))
         self.receive = tactic.SkillEntry(receive.Receive())
-        self.receiver_cost = Receiver_cost(target_point)
-        self.Passer_cost = Passer_cost()
+        self.receiver_cost = ReceiverCost(target_point)
+        self.Passer_cost = PasserCost()
```
Comment (kfu02) [inline rj_gameplay/rj_gameplay/tactic/pass_tactic.py:65]:
```suggestion
        self.passer_cost = PasserCost()
```

**Review by kfu02 — APPROVED** [review]
Sim looks solid. Two minor nits to clean up but otherwise looks good.

**Pull request merged**